docs/scripting-doc/index.d.ts

```diff
@@ -1064,6 +1064,31 @@ declare class TiledObject {
   resolvedProperties(): TiledObjectProperties;
 }
 
+/**
+ * A Tiled project file primarily defines the list of folders containing
+ * the assets belonging to that project.
+ */
+declare class Project {
+/**
+ * A project-specific directory where you can put Tiled extensions.
+ *  It defaults to simply extensions, so when you have a directory called “extensions” alongside
+ *  your project file it will be picked up automatically.
+ *  The directory is loaded in addition to the global extensions.
+ */
+  extensionsPath: string;
+  /**
+   * Path to a file where object custom types are stored.
+   */
+  objectTypesFile: string;
```

Comment:
Ah, you'll find this property is always an empty string, and there is no way to set it in the UI anymore, so this can be removed.

It only exists for compatibility reasons. Any types referenced here will be loaded and merged into `project.propertyTypes` in `ProjectManager::setProject`, after which the property is cleared.

Reply:
Okay. So i'll remove it from the index.d.ts and from the c++ code.